Sources/bip32-ed25519-swift/bip32-ed25519-swift.swift

```diff
@@ -167,7 +176,43 @@ public class Bip32Ed25519 {
         return (z, childChainCode)
     }
 
-    func deriveChildNodePrivate(extendedKey: Data, index: UInt32) -> Data {
+    func deriveChildNodePublic(extendedKey: Data, keyIndex: UInt32, g: Int = 9) throws -> Data {
```

Comment:
Could you call it simply index (i in the paper)? We call this with change as well

Reply:
Sure thing 👍

Reply:
Also instead of g: Int = 9 could it be be g = BIP32DerivationType.Peikert